ui_framework/src/components/call_out/_call_out.scss

```diff
@@ -0,0 +1,57 @@
+.kuiCallOut {
+  padding: $kuiSizeM $kuiSizeL;
+  border-left: 4px solid $kuiColorPrimary;
+  background-color: rgba($kuiColorPrimary, 0.1); /* 1 */
```

Comment:
> It also makes it easier to scan the modifiers and know exactly what kind of use cases a component addresses.

Nesting modifiers to introduce more specificity is definitely an option (which I'm not opposed to), but another benefit of my suggestion is improved maintainability.